perun-core/src/test/java/cz/metacentrum/perun/core/api/AuthzResolverIntegrationTest.java

```diff
@@ -39,6 +39,29 @@
 	final ExtSource extSource = new ExtSource(0, "AuthzResolverExtSource", ExtSourcesManager.EXTSOURCE_LDAP);
 	private int userLoginSequence = 0;
 
+	@Test
+	public void unauthorizedPerunAdmin() throws Exception {
+		assertFalse(AuthzResolver.authorized(new PerunSessionImpl(
+			perun,
+			new PerunPrincipal("pepa", ExtSourcesManager.EXTSOURCE_NAME_INTERNAL, ExtSourcesManager.EXTSOURCE_INTERNAL),
+			new PerunClient()
+		), "default_policy"));
```

Comment:
I really appreciate the code style used here. It makes it really easy to read. :slightly_smiling_face:

Reply:
Copy-paste. Credit goes to @ondrejvelisek :)